.env.sample

```
@@ -122,4 +122,6 @@ VITE_GATEWAY=HTTP
 #GATEWAY
 #IMMERSION_FACILE_BASE_URL
 #POSTGRES_PORT
-#EXPOSED_PORT
\ No newline at end of file
+#EXPOSED_PORT
+
+EMAIL_DOMAINS_BLACKLIST=domain.com
```

est-ce qu'on peut avoir un exemple avec plusieurs domain ? Histoire d'avoir la syntax du séparateur.
```suggestion
EMAIL_DOMAINS_BLACKLIST=domain1.com,domain2.com
```